packages/frontend-2/components/viewer/resources/ModelCard.vue

```diff
@@ -40,11 +43,21 @@
         </div>
         <div
           v-else
-          class="flex flex-none items-center space-x-2 text-xs font-bold opacity-0 transition-opacity group-hover:opacity-100"
+          :class="`${
+            showVersions ? 'text-white' : ''
```

Comment:
Validate if any non-semantic colors really can't be replaced by semantic ones and if they look good in all themes